### PR TITLE
Add missing doc/comment for `leeway` in scheduler

### PR DESCRIPTION
**Pull request by Cee**
Also using the latest Swift preferred comment format. :)

### CODE REVIEW

File: RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift [hunk 1]
```diff
@@ -16,23 +16,28 @@ import Dispatch
...
 
     let configuration: DispatchQueueConfiguration
-    
+
     /// Constructs new `ConcurrentDispatchQueueScheduler` that wraps `queue`.
     ///
-    /// - parameter queue: Target dispatch queue.
+    /// - Parameters:
+    ///   - queue: Target dispatch queue.
+    ///   - leeway: The amount of time, in nanoseconds, that the system can defer the timer.
```
Comment (freak4pc) [inline RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift]: is it "can" defer, or "will" defer ?

File: RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift [hunk 2]
```diff
@@ -44,38 +49,35 @@ public class ConcurrentDispatchQueueScheduler: SchedulerType {
...
     }
 
-    /**
-    Schedules an action to be executed immediately.
-    
-    - parameter state: State passed to the action to be executed.
-    - parameter action: Action to be executed.
-    - returns: The disposable object used to cancel the scheduled action (best effort).
-    */
+    /// Schedules an action to be executed immediately.
```
Comment (freak4pc) [inline RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift]: Any reason you changed all comment blocks from 
```
/**

*/
```
To
``` 
///
///
///
```

Please leave the existing style as that is the style used in all source files in the framework.

File: RxSwift/Schedulers/SerialDispatchQueueScheduler.swift [hunk 3]
```diff
@@ -27,41 +27,46 @@ In case some customization need to be made on it before usage,
 internal serial queue can be customized using `serialQueueConfiguration`
 callback.
 */
-public class SerialDispatchQueueScheduler : SchedulerType {
+public class SerialDispatchQueueScheduler: SchedulerType {
     public typealias TimeInterval = Foundation.TimeInterval
     public typealias Time = Date
-    
-    /// - returns: Current time.
```
Comment (freak4pc) [inline RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: Please revert this change. `returns: Something` is the correct syntax Xcode detects.

File: RxSwift/Schedulers/SerialDispatchQueueScheduler.swift [hunk 4]
```diff
@@ -27,41 +27,46 @@ In case some customization need to be made on it before usage,
...
     }
 
     let configuration: DispatchQueueConfiguration
-    
+
+    /// Constructs new `SerialDispatchQueueScheduler` that wraps `queue`.
+    ///
+    /// - Parameters:
+    ///   - serialQueue: Target dispatch queue.
+    ///   - leeway:
```
Comment (freak4pc) [inline RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: Missing explanation for leeway

**Comment by freak4pc** [comment]
PS There is no "preferred" documentation format from any official entity such as Apple etc. So I don't see a reason making this change. Both options are valid, but better to keep the style already across the framework.

**Comment by Cee** [comment]
@freak4pc Okay, let's revert that.

**Comment by Cee** [comment]
@freak4pc When you click "Editor -> Structure -> Add Documentation" or press <kbd>Command</kbd>+<kbd>Option</kbd>+<kbd>/</kbd>, Xcode would generate `///` one rather than the old comment `/* ... */` block.

File: RxSwift/Schedulers/SerialDispatchQueueScheduler.swift [hunk 5]
```diff
@@ -38,6 +38,11 @@ public class SerialDispatchQueueScheduler : SchedulerType {
 
     let configuration: DispatchQueueConfiguration
     
+    /// Constructs new `SerialDispatchQueueScheduler` that wraps `queue`.
+    ///
+    /// - Parameters:
+    ///   - serialQueue: Target dispatch queue.
```
Comment (freak4pc) [inline RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: This has to be the same format as others: 

- parameter serialQueue: description ...
- parameter leeway: text...

**Comment by freak4pc** [comment]
About the comment-style, it's very interesting. Some files inside the project use different kinds of comment format which is super strange. 

Perhaps @kzaher can have his opinion. It's relatively minor but better to have everything aligned to one style.

**Comment by Cee** [comment]
Comparsion: 
https://github.com/Cee/RxSwift/blob/9daf8e437136c1486b2ed6391bb91e86b8b0840a/RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift#L26-L29
and
https://github.com/Cee/RxSwift/blob/9daf8e437136c1486b2ed6391bb91e86b8b0840a/RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift#L49-L55

IDK currently we are strictly following one of these formats. I'd like to use Xcode generated one, which is `///`.

File: RxSwift/Schedulers/SerialDispatchQueueScheduler.swift [hunk 6]
```diff
@@ -38,6 +38,12 @@ public class SerialDispatchQueueScheduler : SchedulerType {
 
     let configuration: DispatchQueueConfiguration
     
+    /**
+    Constructs new `SerialDispatchQueueScheduler` that wraps `queue`.
```
Comment (freak4pc) [inline RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: One last nit-pick, it should be "that wraps `serialQueue`". I think I'm good beyond that.

Comment (Cee) [reply RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: @freak4pc What about https://github.com/Cee/RxSwift/blob/9daf8e437136c1486b2ed6391bb91e86b8b0840a/RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift#L26

Comment (freak4pc) [reply RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: The argument is `queue` in that link. Here its `serialQueue` that's why I mentioned it.

Comment (Cee) [reply RxSwift/Schedulers/SerialDispatchQueueScheduler.swift]: oh okay then

**Comment by freak4pc** [comment]
Thanks for your patience @Cee !

**Comment by Cee** [comment]
@freak4pc Thank you for your nice catches!
